internal/pkg/errors/errorHTTPResponse.go

```diff
@@ -0,0 +1,22 @@
+package errors
```

Comment:
Shouldn't this package be in the external packages as it doesn't contain any specific logic related to the project?